src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -1602,6 +1610,10 @@
             return _classes;
         },
 
+        toISO: function (date) {
```

Comment:
JSDoc this method what it returns and why you are only returning first 10 chars.

Reply:
Maybe call it `toISOString(date)`

Reply:
> Maybe call it `toISOString(date)`

I'll rename it to `toISODateString` and add documentation. Basically we just care about the date part here and not about the time, so I'm taking only the date part.